include/nuttx/sensors/bmi160.h

```diff
@@ -93,12 +93,14 @@ struct gyro_t
   int16_t z;
 };
 
+#ifdef CONFIG_SENSORS_BMI160_CHARACTER_DEVICE
```

Comment:
let's add bmi160_uorb.h and bmi160_uorb.c to avoid change the char driver.

Reply:
Done